### PR TITLE
Downgrades Mongoose to solve problem registring devices in MongoDB

### PR DESCRIPTION
**Pull request by dcalvoalonso**
This PR tries solves issues #131 downgrading Mongoose.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "coap-cli": "0.4.0",
     "command-node": "0.1.1",
     "mongodb": "2.2.35",
-    "mongoose": "4.13.12",
```
Comment (fgalan) [inline package.json]: Line in CNR:

```
- Update MongoDB and Mongoose version to 2.2.35 and 4.13.12 respectively
```

should be adjusted.

Comment (dcalvoalonso) [reply package.json]: Fixed with 5eef71c1472478fae00da8de262b68b6173c34b5

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -5,3 +5,4 @@
 - Fix: Avoid faster libcoap response: libcoap is not waiting long enough for lightweight-iotagent and lwm2m-node-lib to send the response (#104)
 - Fix: cannot handle update registration from a Wakaama client (#106)
 - Update COAP librery to version 0.21.0
+- Update MongoDB and Mongoose version to 2.2.35 and 4.13.12 respectively
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Mongo has been set to 4.4.3 in this PR... 4.13.12 in CNR is not correct.

In fact, if this PR is going back to the situation previous to #129 I think the right line should be just:

```
- Update MongoDB to 2.2.35
```

Comment (dcalvoalonso) [reply CHANGES_NEXT_RELEASE]: Fixed with 0b6c08ad873b55f263a7a528fcb1fac2982bf1ab

**Comment by fgalan** [comment]
@dcalvoalonso please solve the conflics in the PR so I can merge :)

**Comment by dcalvoalonso** [comment]
Conflicts solved with 83a10d1caf987b5e6b20e930bdd76183c228cb5c :)

**Pull request merged**